File: biometric/src/main/java/dev/skomlach/biometric/compat/engine/internal/face/facelock/IFaceLockCallback.java
package dev.skomlach.biometric.compat.engine.internal.face.facelock;

import androidx.annotation.RestrictTo;

@RestrictTo(RestrictTo.Scope.LIBRARY)
public interface IFaceLockCallback {

    void unlock() throws android.os.RemoteException;

    void cancel() throws android.os.RemoteException;

    void reportFailedAttempt() throws android.os.RemoteException;

    void exposeFallback() throws android.os.RemoteException;

    void pokeWakelock() throws android.os.RemoteException;
}

File: biometric/src/main/java/dev/skomlach/biometric/compat/engine/internal/face/facelock/IFaceLockCallback.kt
package dev.skomlach.biometric.compat.engine.internal.face.facelock

import android.os.RemoteException
import androidx.annotation.RestrictTo

@RestrictTo(RestrictTo.Scope.LIBRARY)
interface IFaceLockCallback {
    @Throws(RemoteException::class)
    fun unlock()

    @Throws(RemoteException::class)
    fun cancel()

    @Throws(RemoteException::class)
    fun reportFailedAttempt()

    @Throws(RemoteException::class)
    fun exposeFallback()

    @Throws(RemoteException::class)
    fun pokeWakelock()
}